test/relations.integration.js

```diff
@@ -1652,7 +1652,7 @@ describe('relations - integration', function() {
           expect(res.body).to.be.an('object');
           expect(res.body.error).to.be.an('object');
           expect(res.body.error.name).to.equal('Error');
-          expect(res.body.error.statusCode).to.equal(500);
+          expect(res.statusCode).to.equal(500);
```

Comment:
@bajtos : it was causing the test cases to fail on local. and I guess, it's the same reported by @loay in his [PR#2411](https://github.com/strongloop/loopback/pull/2411/commits/32318ec990f05ab638de811717c3e8637a57c8fa#diff-295cd28b918f0a0d539d8fd920b5c1f2R1655).

I will remove it from the context of this commit.